### ISSUE TITLE
Vuepress vulnerabilities

### ISSUE THREAD

**Issue by yves-chevallier** [heig-tin-info/baygon#19]
```
remote: Resolving deltas: 100% (6/6), completed with 6 local objects.
remote: 
remote: GitHub found 11 vulnerabilities on heig-tin-info/baygon's default branch (4 high, 5 moderate, 2 low). To find out more, visit:
remote:      https://github.com/heig-tin-info/baygon/security/dependabot
remote: 
```

**Comment by yves-chevallier** [comment]
Perhaps migrate to Nuxt?

**Comment by yves-chevallier** [comment]
Fix in [5b9e3e8](https://github.com/heig-tin-info/baygon/commit/5b9e3e869d47964552a7d7614bdad2d9530a9e14)
